tests/project/_common/test_movement.py

```diff
@@ -0,0 +1,437 @@
+
+    # get artifact path
+    description = op.description
+    artifact_path = str(description).split(" ")[-1]
+    print(artifact_path)
+
+    assert artifact_path[0] == "/"
+    assert artifact_path[-4:] == ".zip"
+
+    # get existing project
+    existing_project = client.projects.by_name("minimal_incomplete_categorization")
+
+    # test trying to set new_project name on existing project
+    with pytest.raises(KeyError):
+        categorization.import_artifacts(
+            tamr_client=existing_project.client,
+            project_artifact_path=artifact_path,
+            target_project=existing_project,
+            new_project_name="new_project_name",
+            overwrite_existing=True,
+        )
+
+    # test trying to set new_unified_dataset_name on existing project
+    with pytest.raises(KeyError):
+        categorization.import_artifacts(
+            tamr_client=existing_project.client,
+            project_artifact_path=artifact_path,
+            target_project=existing_project,
+            new_unified_dataset_name="new_ud_name",
+            overwrite_existing=True,
+        )
+
+    # test overwrite existing
+    with pytest.raises(KeyError):
+        categorization.import_artifacts(
+            tamr_client=existing_project.client,
+            project_artifact_path="incorrect/artifact/path",
+            target_project=existing_project,
+            overwrite_existing=False,
+        )
+
+    # test incorrect artifact path
+    with pytest.raises(ValueError):
+        categorization.import_artifacts(
+            tamr_client=existing_project.client,
+            project_artifact_path="incorrect/artifact/path",
+            target_project=existing_project,
+            overwrite_existing=True,
+        )
+
+    # fail if not present
+    with pytest.raises(ValueError):
+        categorization.import_artifacts(
+            project_artifact_path=artifact_path,
+            tamr_client=client,
+            target_project=existing_project,
+            include_destructive_artifacts=[catfacts.CATEGORIZATION_TAXONOMIES],
+            fail_if_not_present=True,
+            overwrite_existing=True,
+        )
+
+    # test incorrect artifact name
+    with pytest.raises(ValueError):
+        categorization.import_artifacts(
+            tamr_client=existing_project.client,
+            project_artifact_path=artifact_path,
+            target_project=existing_project,
+            include_additive_artifacts=["incorrect_artifact_name"],
+            overwrite_existing=True,
+        )
+    with pytest.raises(ValueError):
+        categorization.import_artifacts(
+            tamr_client=existing_project.client,
+            project_artifact_path=artifact_path,
+            target_project=existing_project,
+            include_destructive_artifacts=["incorrect_artifact_name"],
+            overwrite_existing=True,
+        )
+    # Expected ValueError but got RuntimeError instead
+    with pytest.raises(RuntimeError):
```

Comment:
What is this testing? Appears to be the same as call as the previous one

Reply:
subtle difference: include_destructive_artifacts vs include_additive_artifacts. Both are tested incorrect artifact names in list.